docker-compose.yml

```
@@ -27,7 +27,7 @@ services:
     build: docker/grafana
     restart: always
     ports:
-      - "3000:3000"
+      - "127.0.0.1:3000:3000"
```

This file is not for local development, no need to bind local

```suggestion
      - "3000:3000"
```